warehouse/models/intermediate/gtfs/int_gtfs_schedule__long_calendar.sql

```diff
@@ -0,0 +1,30 @@
+-- TODO: make an intermediate calendar and use that instead of the dimension
+WITH dim_calendar AS (
+    SELECT *
+    FROM {{ ref('dim_calendar') }}
+),
+
+int_gtfs_schedule__long_calendar AS (
+    -- Note that you can unnest values easily in SQL, but getting the column names
+    -- is weirdly hard. To work around this, we just UNION ALL.
```

Comment:
just curious had you tried https://cloud.google.com/bigquery/docs/reference/standard-sql/query-syntax#unpivot_operator?

Reply:
This is copy-pasted from our existing long calendar table, didn't want to reinvent the wheel: https://github.com/cal-itp/data-infra/blob/main/warehouse/models/gtfs_views/gtfs_schedule_stg_calendar_long.sql. Can add a TODO comment

Reply:
https://github.com/cal-itp/data-infra/pull/1936/commits/acab7027b6ec5f1bc6ac5fac1fa2230b3c174557